addons/mail/static/src/models/rtc_peer_connection.js

```diff
@@ -20,8 +21,40 @@ registerModel({
             const transceivers = this.peerConnection.getTransceivers();
             return transceivers[this.messaging.rtc.orderedTransceiverNames.indexOf(trackKind)];
         },
+        /**
+         * The download is allowed when there are views that display the video stream.
+         *
+         * @private
+         * @returns {boolean}
+         */
+        _computeAcceptsVideoStream() {
+            return Boolean(this.rtcSession.callParticipantCards && this.rtcSession.callParticipantCards.length > 0);
+        },
+        /**
+         * @private
+         */
+        _onChangeAcceptsVideoStream() {
+            const transceiver = this.getTransceiver('video');
+            if (!transceiver) {
+                return;
+            }
+            const rtc = this.rtcSession.rtcAsConnectedSession;
+            if (this.acceptsVideoStream) {
+                transceiver.direction = rtc.videoTrack ? 'sendrecv' : 'recvonly';
+                console.log(`download from ${this.rtcSession.name}: allowed (direction: ${transceiver.direction})`); // TODO remove
+            } else {
+                transceiver.direction = rtc.videoTrack ? 'sendonly' : 'inactive';
+                console.log(`download from ${this.rtcSession.name}: disallowed (direction: ${transceiver.direction})`); // TODO remove
+            }
```

Comment:
At a glance the value of `transceiver.diection` depends on `RtcPeerConnection/acceptsVideoStream` and also `Rtc/videoTrack`, but `Rtc/videoTrack` is not listed in `dependencies` of this `OnChange`.

I think you should include it too.